### ISSUE TITLE
What does auxiliary feat mean?

### ISSUE THREAD

**Issue by ZhongxingRen** [ByteDance-Seed/Depth-Anything-3#75]
Good job !! 
I can see the potential of upgrade dimension (from 2D to 3D) through your work, which will make a great progress on 3d reconstruction.
Being excited, I read the Depth-Anything-3 code and have some questions that I am curious about: 

1, In most of the cases, you have used DINO v2 as the backbone to generate the feature from images. I wonder that have you frozen the backbone while training DA3 model ?
2, When compared to VGGT which is also a DINO based model, DA3 outperforms it and what kind of tricks you have applied when you design the DPT head and perform the training ? 
3, Concretely, in da3.py, I still don't quite understand the usage of auxiliary feature since it is not mentioned in your paper and with few comments in your code. Is the auxiliary feature from the distinct layers (determined by the input parameter "export_feat_layers" parameter ) of DINO backbone? If I want to get the features from distinct layers in DPT head, is there an api for me to call ?

**Comment by haotongl** [comment]
@ZhongxingRen Hi, thanks for your attention.
1. It's not frozen.
2. VGGT actually uses one DINO model and an additional transformer model (about twice the size of DINO), whereas our method uses only one DINO model.
3. The auxiliary_feature argument is used to visualize the transformer's layers. If you want to extract features from specific layers, you can set:
`export_feat_layers="19,27,33,39"`
https://github.com/ByteDance-Seed/Depth-Anything-3/blob/main/src/depth_anything_3/configs/da3-giant.yaml#L13

**Comment by ZhongxingRen** [comment]
Thanks for your reply,
1，Are these feature and auxiliary_feature argument from DINO v2 backbone?
2，And what is the difference when I get the auxiliary features from different layers? 
If I wanna get the depth feature (not the predicted depth map) for some other downstream tasks, is it possible to get these depth feature inside the depth head (https://github.com/ByteDance-Seed/Depth-Anything-3/blob/main/src/depth_anything_3/model/da3.py#L134)? 

3，When I try to load model.safetensor of "da3-giant" after using api:

model = DepthAnything3(da3-giant)
model.load_state_dict(giant_weights)

it occur the issue:

Missing key(s) in state_dict: "model.head.scratch.output_conv2_aux.1.2.weight", "model.head.scratch.output_conv2_aux.1.2.bias",
"model.head.scratch.output_conv2_aux.2.2.weight", "model.head.scratch.output_conv2_aux.2.2.bias", "model.head.scratch.output_conv2aux.3.2weight",
"model.head.scratch.output_conv2_aux.3.2.bias"

Will the absence of these weights affect the results of getting the auxiliary features?
